build/csv/csv-gen.sh

```diff
@@ -53,6 +53,14 @@ function generate_csv() {
     "${YQ_CMD_WRITE[@]}" "$CSV_FILE_NAME" metadata.annotations.externalClusterScript "$(base64 <$CEPH_EXTERNAL_SCRIPT_FILE)"
     "${YQ_CMD_WRITE[@]}" "$CSV_FILE_NAME" metadata.name "rook-ceph-operator.v${CSV_VERSION}"
 
+    if [[ -n $SKIP_RANGE ]]; then
```

Comment:
Will SKIP_RANGE and REPLACES_CSV_VERSION have different settings in different downstream builds?

Reply:
Yes, the values will get updated for each release and are managed by the downstream build.